tests/plugin/test_broken_plugin.py

```diff
@@ -0,0 +1,30 @@
+from textwrap import dedent
+
+import pytest
+
+
```

Comment:
as discussed let's add test case `test_broken_implementation_plugin` which will try to load a plugin which raises some error during its module loading

Reply:
Named it `failing_plugin`